src/style.css

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  --text-color: #000;
+  --text-color-active: #00acdc;
+  --header-height: 60px;
+}
+
+.visually_hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  border: 0;
+  padding: 0;
+  white-space: nowrap;
+  clip-path: inset(100%);
+  clip: rect(0 0 0 0);
+  overflow: hidden;
```

Comment:
You can just use `display:none` or `visibility: hidden` to hide the element
https://www.freecodecamp.org/news/css-display-none-and-visibility-hidden-the-difference/